src/aiperf/dataset/loader/base_hf_dataset.py

```
@@ -103,6 +107,28 @@ def _extract_videos(self, row: dict[str, Any], video_column: str) -> list[Video]
+        if not isinstance(value, dict):
+            return []
+        array = value.get("array")
+        sr = value.get("sampling_rate")
+        if array is None or sr is None:
+            return []
+        try:
+            buf = io.BytesIO()
+            sf.write(buf, array, sr, format="WAV")
+            b64 = base64.b64encode(buf.getvalue()).decode("utf-8")
+            return [Audio(name="", contents=[f"wav,{b64}"])]
+        except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)